i6_models/parts/e_branchformer/merge.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+__all__ = ["MergerV1Config", "MergerV1"]
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class MergerV1Config(ModelConfiguration):
+    """
+    Attributes:
+        input_dim: input dimension
+        kernel_size: kernel size of the depthwise convolution layer
+        dropout: dropout probability
+    """
+
+    input_dim: int
+    kernel_size: int
+    dropout: float
+
+    def __post_init__(self):
+        super().__post_init__()
```

Comment:
```suggestion
```
If you just call the same function of the parent class, then this code is not necessary. If you remove it, the function is just inherited.